### PR TITLE
Using a screwdriver / wrench on a credit transfer devices will now unanchor them.

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Using a screwdriver on a credit transfer devices will now unanchor them.

## Why's this needed? 
Not being able to manipulate items isn't great and currently you can only smash it completely but only with a wrench which is just bizzare.

## Changelog

```changelog
(u)Carbadox
(+)Using a screwdriver / wrench on a credit transfer devices will now unanchor and let you move them.
```

### CODE REVIEW

**Comment by glassofmilk** [comment]
Wrench is normally the tool for anchoring/unanchoring - I think it might make more sense to use a wrench for this instead of a screwdriver.

**Comment by Carbadox** [comment]
> 
> 
> Wrench is normally the tool for anchoring/unanchoring - I think it might make more sense to use a wrench for this instead of a screwdriver.

The item already uses wrenches, but only for destroying the item for whatever reason. I kept it there in case someone would say it's best for it to stay there. But if people say otherwise, I can make it use wrenching and screwing instead

**Comment by MatheusIsNotAvailable** [comment]
Some stuff uses both wrenches and screwdrivers like the plant trays, dosen't really matter wich one you pick imo

**Comment by BatElite** [comment]
For consistency's sake you can replace the wrenching thing with the anchoring code. I kinda feel 1/3 chance to break it is superfluous when you can use a deconstructor.

File: code/obj/machinery/cashreg.dm [hunk 1]
```diff
@@ -16,6 +16,15 @@
...
 			W = W:ID_card
+		if(istool(W, TOOL_SCREWING))
+			if(!src.anchored)
+				user.visible_message("<b>[user]</b> secures the [src] to the floor!")
+				playsound(src.loc, "sound/items/Screwdriver.ogg", 100, 1)
+				src.anchored = 1
+			else
+				user.visible_message("<b>[user]</b> unbolts the [src] from the floor!")
+				playsound(src.loc, "sound/items/Screwdriver.ogg", 100, 1)
+				src.anchored = 0
```
Comment (ZeWaka) [inline code/obj/machinery/cashreg.dm:27]:
```suggestion
			user.visible_message("<b>[user]</b> [anchored ? "unbolts the [src] from" : "secures the [src] to"] the floor.")
			playsound(src.loc, "sound/items/Screwdriver.ogg", 80, 1)
			src.anchored = !src.anchored
```

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**